webapp/public/javascripts/controllers.js

```diff
@@ -234,18 +246,24 @@ lunchrControllers.controller(
                                             labelClass: "labels",
                                             animation: google.maps.Animation.DROP
                                         },
-                                        click: function (e) {
-                                            var index = $scope.selectedPlaces.indexOf(e.$id);
+                                        click: function( ) {
+                                            var errorAlert = $('#error');
+                                            if(errorAlert.html()){
+                                                errorAlert.html("");
```

Comment:
Precisely. The user had no places selected to then the error was thrown. Now with this code, if there is that error shown, and they click a place to add to the list, it disappears :). as opposed to them having to see it until they navigate away from the page.